README.md

```diff
@@ -1 +1,180 @@
-# TDD-project
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [x] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍